### PR TITLE
Update securing_builds doc with warning on networking rule bypass

### PR DESCRIPTION
**Pull request by sfowl**
Proposed warning only applies to OpenShift 3.x, not OpenShift 4.

### CODE REVIEW

File: admin_guide/securing_builds.adoc
Comment (juanluisvaladas) [inline admin_guide/securing_builds.adoc]: The warning is completely fine but perhaps we should be more explicit saying this only affects docker builds and potentially custom builds to avoid customers being concerned about non real problems.
Could  we replace "During builds" with "During docker and custom builds"?

Comment (sfowl) [reply admin_guide/securing_builds.adoc]: Thanks, fixed.

**Review by juanluisvaladas — APPROVED** [review]
lgtm

**Comment by vikram-redhat** [comment]
Needs to be CP'ed to master-3. Not to be merged till QE has acked it.

**Review by kalexand-rh — COMMENTED** [review]
LGTM, pending QE approval.

File: admin_guide/securing_builds.adoc
Comment (jboxman) [inline admin_guide/securing_builds.adoc]: Would this make more sense as:

> networking rules including egress network policy...

Or

> networking rules including those defined by EgressNetworkPolicy objects...

**Comment by jboxman** [comment]
@sfowl, can you squash the commit and force-push? Thanks!

File: admin_guide/securing_builds.adoc
Comment (sfowl) [reply admin_guide/securing_builds.adoc]: Thanks, went with the latter.

**Comment by sfowl** [comment]
> @sfowl, can you squash the commit and force-push? Thanks!

Yep, done.

**Pull request merged**
